tests/pkg_tinydtls_sock_async/dtls-server.c

```diff
@@ -105,9 +106,9 @@ static void _dtls_handler(sock_dtls_t *sock, sock_async_flags_t type, void *arg)
         return;
     }
 
-    if (type & SOCK_ASYNC_CONN_RECV) {
-        sock_dtls_session_t session = { 0 };
+    sock_dtls_session_t session = { 0 };
 
+    if (type & SOCK_ASYNC_CONN_RECV) {
         puts("Session handshake received");
```

Comment:
Here and in the other event types that are not `SOCK_ASYNC_CONN_FIN` or `SOCK_ASYNC_CONN_RDY` (as well in the client), you could also use `expect(!sock_dtls_get_event_session(sock, &session))` to make this proper tests ;-).

Reply:
Done! In the event types `SOCK_ASYNC_CONN_FIN` and `SOCK_ASYNC_CONN_RDY` we could also use `expect(sock_dtls....)` to test it since currently it is only printed and not tested. Would that be too much to also add it there? Would like to keep the session printing, because it is currently the only example usage of this new function.

Reply:
> Would like to keep the session printing, because it is currently the only example usage of this new function.

Yes, as you print output, that provides enough result (more than expect which does not print anything when succeeding)